src/EnergyPlus/AirflowNetwork/src/Properties.cpp

```diff
@@ -139,13 +158,12 @@ namespace AirflowNetwork {
             T = UpperLimit;
         }
 
-        return airThermConductivity(state, T) / (AIRCP(W) * AIRDENSITY(state, P, T, W));
+        return thermal_conductivity(T) / (AIRCP(W) * Psychrometrics::PsyRhoAirFnPbTdbW(m_state, P, T, W));
     }
 
-    Real64 airPrandtl(EnergyPlusData &state,
-                      Real64 T, // Temperature in Celsius
-                      Real64 W, // Humidity ratio
-                      Real64 P  // Barometric pressure
+    Real64 AirProperties::prandtl_number(Real64 P, // Barometric pressure
+                                         Real64 T, // Temperature in Celsius
+                                         Real64 W  // Humidity ratio
     )
     {
         // Dry air Prandtl number {-}
```

Comment:
This is a person's name? I  thought it was a Fortran style variable.

Reply:
Yep, founding father of the science of aerodynamics. You don't have any of his books? Doesn't everyone? https://en.wikipedia.org/wiki/Ludwig_Prandtl